packages/stockflux-launcher/src/helpers/getUndockedPosition.js

```diff
@@ -0,0 +1,103 @@
+import { OpenfinApiHelpers } from 'stockflux-core';
+
+async function getBounds() {
+  let bounds = await OpenfinApiHelpers.getCurrentWindowSync().getBounds();
+  return bounds;
+}
+
+async function getMonitorInfo() {
+  let monitorInfo = await window.fin.System.getMonitorInfo();
+  return monitorInfo;
+}
```

Comment:
Can we move this into `OpenfinApiHelpers` as that is the only other place within the system which references `window.fin`.